.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @renardete @holgiosalos @Valeryibarra
```

Comment:
To avoid this warning from github:
![image](https://user-images.githubusercontent.com/10209180/181026495-08875aab-27af-4589-ba0a-e3f17b146446.png)

You should add an empty line at the end of the file (or linebreak), that line is called EOF. You can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969 

Next time that we see this, we will comment just EOF as a reminder 😊

Reply:
Thank you for your comment, I have solved this problem!

I see another error, but it is because the other two contributors have not accepted my request.

Reply:
yes, if you look into this guide there are more possible collaborators.
https://github.com/testing-community/cypress-training-doc